src/main/java/com/puppycrawl/tools/checkstyle/filefilters/BeforeExecutionExclusionFileFilter.java

```diff
@@ -74,6 +74,14 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * To configure the filter to ignore some Checks in maven test folder by modifying
+ * {@code config/suppressions.xml}:
+ * </p>
+ *
+ * <pre>
+ * &lt;suppress checks=&quot;EmptyBlock&quot; files=&quot;.*[\\/]src[\\/]test[\\/]&quot;/&gt;
```

Comment:
All examples should be for `* &lt;module name=&quot;BeforeExecutionExclusionFileFilter&quot;&gt;` only

Reply:
Ok i will make a module and put into this